src/sentry/static/sentry/less/group-detail.less

```diff
@@ -44,6 +44,25 @@
     border-radius: 15px;
   }
 
+  .event-message {
+    position: relative;
+
+    .error-level {
```

Comment:
this fixes another issue where a long message forces the error level to wrap on a new line as well

Reply:
Looks like we hide the level-indicator on the issue share page... Digging in now to see if there's a quick fix. https://percy.io/getsentry/sentry/builds/1118989/view/69878739/1280?browser=firefox&mode=diff